manifests/init.pp

```diff
@@ -231,7 +231,7 @@
       path    => "${confdir}/puppet.conf",
       section => 'main',
       setting => 'hiera_config',
-      value   => $hiera_yaml,
+      value   => '$confdir/hiera.yaml',
```

Comment:
Yes that is right. I just wanted that there are no conflicts with other modules. I will set `puppet_conf_manage` to false or use something else. After all, the module does not work properly with Puppet 4.